ddpui/tests/api_tests/test_superset_api.py

```
@@ -0,0 +1,504 @@
+        csrf_response = Mock()
+        csrf_response.json.return_value = {"result": "test-csrf-token"}
+        csrf_response.raise_for_status = Mock()
+        csrf_response.headers = {"Set-Cookie": "session=test-session-cookie"}
+        mock_get.return_value = csrf_response
+
+        warehouse = OrgWarehouse.objects.create(
+            wtype="postgres", credentials="creds", org=orguser_with_viz.org
+        )
+        request = mock_request(orguser_with_viz)
+        result = post_fetch_embed_token(request, "dashboard-uuid")
+        assert result["embed_token"] == "test-embed-token"
```

Ruff (0.15.9)

[error] Possible hardcoded password assigned to: "embed_token"

(S105)